bitmap-plasma/app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
         minSdkVersion 14
         targetSdkVersion 25
         ndk {
-            abiFilters 'armeabi', 'armeabi-v7a', 'arm64-v8a', 'x86'
+            abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86'
```

Comment:
Why not x86_64?

What happens if you don't set this? Won't it default to all? If not, we should change that.

Reply:
It still compiles and runs fines with x86_64 enabled, that would be default for r16b.  set to default works fine.